src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -670,6 +715,18 @@ private function updateData(array $data, string $attribute, mixed $attributeValu
             return $data;
         }
 
+        if ($this->classMetadataFactory && $this->classMetadataFactory->hasMetadataFor($class)) {
+            $classMetadata = $this->classMetadataFactory->getMetadataFor($class);
+            $attributesMetadata = $classMetadata->getAttributesMetadata();
+            if (isset($attributesMetadata[$attribute]) && $serializedPath = $attributesMetadata[$attribute]->getSerializedPath()) {
```

Comment:
`if ($serializedPath = $attributesMetadata[$attribute]?->getSerializedPath()) {`

Reply:
This throws a warning when the array key is undefined, so I kept it as is. Couldn't figure out a clever way to use the null coalescing operater here instead.